### PR TITLE
[WIP] Updating docs about operator and endpoints

### PR DESCRIPTION
**Pull request by metonymic-smokey**
**Description**
This PR is for updating docs about how endpoints work in the `meshconfig` and basics of the operator lifecycle. 

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/concepts/architecture/operator.md [hunk 1]
```diff
@@ -25,4 +25,7 @@ MeshSync Controller manages the lifecycle of MeshSync that is deployed for resou
 
 See the "[MeshSync]({{site.baseurl}}/architecture/meshsync)" for more information.
 
-### Operator FAQs
\ No newline at end of file
+### Operator FAQs
+
+#### When is the operator deployed and when is it deleted?
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/operator.md:30]:
```suggestion
#### When is Meshery Operator deployed and when is it deleted?  
```

**Comment by metonymic-smokey** [comment]
thank you @leecalcote !

**Comment by leecalcote** [comment]
> thank you @leecalcote !

Teamwork! 🙌

**Comment by leecalcote** [comment]
Good stuff here. Thanks for making these updates, @metonymic-smokey

**Pull request merged**